test/tools/invoker.py

```diff
@@ -50,6 +51,18 @@ def cleanup(self):
         if os.path.exists(descriptor_set_output):
             os.remove(descriptor_set_output)
 
+    @classmethod
+    def _get_protoc_binary(cls) -> str:
+        system_to_protoc_binary = {
+            "Linux": "protoc",
+            "Darwin": "osx-protoc",
```

Comment:
Is this because you're developing it on a mac?

Reply:
Yes! But I can remove it and develop on cloudtop using Linux :)

Reply:
Done. Thanks!